operations/iuf/IUF.md

```diff
@@ -434,19 +441,19 @@ These [examples](examples/iuf_restart.md) highlight common use cases of `iuf res
 
 #### `activity`
 
-The `activity` subcommand allows the administrator to create a new activity, display details for an activity, and create, update, and annotate activity states. These operations allow the administrator to easily determine the status
+The `activity` subcommand allows the administrator to create a new activity, display details for an activity, list activities, and create, update, and annotate activity states. These operations allow the administrator to easily determine the status
 of IUF activity operations and associate time-based metrics and user-specified comments with them.
 
 The activity details displayed are:
 
-| Column         | Description                                                      |
-| -------------- | ---------------------------------------------------------------- |
-| Start          | The time that this operation began execution                     |
-| Category       | The state of the activity when the operation was created         |
-| Argo Workflow  | The Argo workflow associated with the operation                  |
-| Status         | The status of the operation                                      |
-| Duration       | How long the operation has been in this state (if not completed) |
-| Comment        | User-specified comments associated with the operation            |
+| Column         | Description                                                          |
+| -------------- | -------------------------------------------------------------------- |
+| Start/session  | The time that this operation began execution and name of session     |
```

Comment:
The column header still says only ```Start``` on gamora.

Reply:
Just updated it in the code.